### ISSUE TITLE
Terrain map like sample app not working

### ISSUE THREAD

**Issue by wiulma** [rnmapbox/maps#2576]
### Mapbox Implementation

Mapbox

### Mapbox Version

default

### Platform

Android

### `@rnmapbox/maps` version

10.0.0-beta.65

### Standalone component to reproduce

```typescript
import { StyleSheet, View } from 'react-native'

const App = () => {
	return (
		<View style={styles.page}>
			<TerrainMap />
		</View>
	)
}

export default App

const styles = StyleSheet.create({
	page: {
		flex: 1,
		justifyContent: 'center',
		alignItems: 'center'
	}
})

import { MapView, SkyLayer, Terrain, Logger, RasterDemSource, Atmosphere, Camera } from '@rnmapbox/maps'
import { useRef } from 'react'

Logger.setLogLevel('verbose')

export const TerrainMap = () => {
	const cameraRef = useRef<Camera>(null)
	return (
		<MapView style={{ flex: 1 }} styleURL={'mapbox://styles/mapbox-map-design/ckhqrf2tz0dt119ny6azh975y'}>
			<Camera
				ref={cameraRef}
				centerCoordinate={[
					// -74.00597, 40.71427
					// -122.4189591, 37.6614238,
					-114.34411, 32.6141
				]}
				zoomLevel={13.1}
				heading={80}
				pitch={85}
			/>
			<RasterDemSource
				id="mapbox-dem"
				url="mapbox://mapbox.mapbox-terrain-dem-v1"
				tileSize={514}
				maxZoomLevel={14}>
				<Atmosphere
					style={{
						color: 'rgb(186, 210, 235)',
						highColor: 'rgb(36, 92, 223)',
						horizonBlend: 0.02,
						spaceColor: 'rgb(11, 11, 25)',
						starIntensity: 0.6
					}}
				/>
				<SkyLayer
					id="sky-layer"
					style={{
						skyType: 'atmosphere',
						skyAtmosphereSun: [0.0, 0.0],
						skyAtmosphereSunIntensity: 15.0
					}}
				/>
				<Terrain style={{ exaggeration: 1.5 }} />
			</RasterDemSource>
		</MapView>
	)
}

```

### Observed behavior and steps to reproduce

I get an error:
````

 ERROR  Invariant Violation: requireNativeComponent: "RCTMGLAtmosphere" was not found in the UIManager.
````

### Expected behavior

I expect with sample is working

### Notes / preliminary analysis

Hi!
I created a new react native project from scratch (with react native cli).
I installed the mapbox dependency:
`````
npm install --save rnmapbox/maps#main
`````
then I follow the android instalaltion guide.
I created the download token, and configure the build.gradle file as explained.

Then, I just copy/past the TerrainSkyAtmosphere in the sample folder, but I get this error:

`````
* What went wrong:
Could not determine the dependencies of task ':rnmapbox_maps:compileDebugAidl'.
> Could not resolve all task dependencies for configuration ':rnmapbox_maps:debugCompileClasspath'.
   > Could not find com.mapbox.maps:android:10.10.0.
     Required by:
         project :rnmapbox_maps
   > Could not find com.mapbox.mapboxsdk:mapbox-sdk-turf:6.8.0.
     Required by:
         project :rnmapbox_maps

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

`````
I also tried to customize the version in the build gradle, but I got the same error.
Do you have any suggestion to make it working?
thanks a lot

**Comment by mfazekas** [comment]
Do you have RNMapboxMapsImpl configured to v10 implementation (mapbox) by default it's maplibre which doesn't have Atmosphere, etc.

**Comment by wiulma** [comment]
Hi! thanks for the quick reply. I configured the build.gradle to use the "mapbox" option for RNMapboxMapsImpl, but it wasn't working.
I update the npm package this morning and I moved the maven repo configuration suggested in the android setup instructions from the buildscript repo configuration object to the allprojects configuration one. 
I don't really know if this did the trick (or because I cleaned the local cache again and again) but now it's working. 
Thanks for your support
